code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -143,26 +143,6 @@
 	icon_state = "deaglecamo"
 	item_state = "deagleg"
 
-/*
-/obj/item/weapon/gun/projectile/fiveseven

```

Comment:
whoops I deleted this commented out text and forgot about it

whatever it was a gun I added to begin with and then nobody wanted because snowflake bullets

Reply:
do not remove my good belgian weapons
i will work with my honey some day.

Reply:
You can put it back then. Right now it's just clutter I forgot to delete from my own gun update.

Reply:
Alright. Fair enough.

Reply:
Like I'm pretty sure I've said before: Focus your PRs please. If this is a sound-related PR, please keep it to sound-related changes.

Reply:
This was accidental, I assure you. I had several separate PRs you'll notice because I was trying to avoid this happening.

Reply:
Please separate your PRs, it keeps my headaches down.